pyupgrade/_plugins/lru_cache.py

```diff
@@ -21,6 +22,10 @@ def _remove_call(i: int, tokens: List[Token]) -> None:
     del tokens[i:j + 1]
 
 
+def _replace_lru_cache_with_cache(i: int, tokens: List[Token]) -> None:
+    find_and_replace_call(i, tokens, template='functools.cache')
```

Comment:
use `functools.partial` like the other files do, no need for a separate function